src/rcl_timer.c

```diff
@@ -30,8 +30,11 @@ ERL_NIF_TERM nif_rcl_timer_init(ErlNifEnv *env, int argc, const ERL_NIF_TERM arg
   rcl_ret_t rc;
   rcl_timer_t timer         = rcl_get_zero_initialized_timer();
   rcl_allocator_t allocator = get_nif_allocator();
-
+#ifdef ROS_DISTRO_jazzy
+  rc = rcl_timer_init2(&timer, clock_p, context_p, RCL_MS_TO_NS(period_ms), NULL, allocator, true);
+#else
   rc = rcl_timer_init(&timer, clock_p, context_p, RCL_MS_TO_NS(period_ms), NULL, allocator);
+#endif
```

Comment:
rcl_timer_init has been deprecated in Jazzy.

ref. https://docs.ros.org/en/jazzy/p/rcl/generated/function_timer_8h_1a799648220ed0f96ea5623853baf27a03.html